### ISSUE TITLE
Sync storage on OneDrive on Windows

### ISSUE THREAD

**Issue by rekire** [juliansteenbakker/flutter_secure_storage#667]
I'm thinking about how to sync settings like on Apple-Platforms with the keychain. I think the OneDrive could be the way to go. The (personal) OneDrive path can be found in the environment variable `OneDrive`. It seems that the most correct way would be to use the "Apps" sub folder which seems to be localized on de_DE it is "Anwendungen".

That Apps folder is a ["special folder"](https://learn.microsoft.com/onedrive/developer/rest-api/api/drive_get_specialfolder?view=odsp-graph-online#special-folder-names) like I found out. However this seems to be an exclusive Windows Graph API which does not make sense in manner of a transparent sync when you need to sign in.

What do you think about it? Is it worth to open a PR for that? Has anyone an idea how to get the special folder path without the REST-API?

**Comment by juliansteenbakker** [comment]
I am closing all older issues. If this issue still exists in the latest version, please let me know.

**Comment by rekire** [comment]
@juliansteenbakker I quickly checked the windows code. There is no one drive support. I understand why you close old issues, but you might should close them a bit more careful.

TL;DR: Please reopen
